src/components/page.js

```diff
@@ -180,7 +180,7 @@ function Page({ children, fullScreen, headerActions, alerts }) {
 
 Page.propTypes = {
   alerts: PropTypes.arrayOf(PropTypes.shape({
-    message: PropTypes.node.isRequired,
+    message: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
```

Comment:
`PropTypes.node` means: anything than can be rendered (string, number, react element etc.)